l10n_br_fiscal/views/tax_definition_view.xml

```diff
@@ -46,13 +45,14 @@
         <sheet>
           <group>
             <field name="tax_group_id"/>
+            <field name="type_in_out"/>
```

Comment:
This field must be invisible, this is used only to define CST field (input or output) from tax

Reply:
I put it there to improve usability when creating an incoming tax definition. I thought about passing it in the context as a `default_type_in_out` entry, but the model `l10n_br_fiscal.tax.definition` is used in many system configuration models, and some don't have this in/out differentition, the company model being an example.

Reply:
@ gabrielcardoso21,

Most tax definitions are defined for outputs as tax definitions on res.company, icms and etc ... on these objects you will not define input tax definition, because it makes no sense ...

Generally when the tax definition is defined as input, it is in a tax operation line and in this case the context of the tax operation is considered, so the field is invisible and must remain invisible.